pkg/vault/contracts/CompositeLiquidityRouterHooks.sol

```diff
@@ -43,13 +43,16 @@ contract CompositeLiquidityRouterHooks is BatchRouterCommon {
         bool isStaticCall;
     }
 
+    bool internal immutable _isAggregator;
```

Comment:
I think we've discussed this before, and clearly we're committed to the "Aggregator" name for the contracts, but should the internal flag be more specific; e.g., assumesPrepayment? The default value should be false for the "normal" router.